conda_build_config.yaml

```diff
@@ -232,21 +132,662 @@ cross_compiler_target_platform:
+blosc:
+  - '1'
+boost:
+  - 1.82
+boost_cpp:
+  - 1.82
+brunsli:
+  - '0'
+bzip2:
+  - '1'
+c_ares:
+  - '1'
+c_blosc2:
+  - '2'
+cairo:
+  - '1'
+capnproto:
+  - 1.1.0
+ceres_solver:
+  - '2.2'
+cfitsio:
+  - '3.470'
+charls:
+  - '2.2'
+coin_or_cbc:
+  - '2.10'
+coin_or_cgl:
+  - '0.60'
+coin_or_clp:
+  - '1.17'
+coin_or_osi:
+  - '0.108'
+coin_or_utils:
+  - '2.11'
+cudnn:
+  - '9'
+cutensor:
+  - '2'
+cyrus_sasl:
+  - 2.1.28
+dav1d:
+  - 1.2.1
+dbus:
+  - '1'
+eigen:
+  - 3.3.7
+epoxy:
+  - '1.5'
+expat:
+  - '2'
+ffmpeg:
+  - '6'
+fftw:
+  - '3.3'
+flatbuffers:
+  - 24.3.25
+fmt:
+  - '9'
+fontconfig:
+  - '2'
+freeglut:
+  - '3'
+freetds:
+  - '1'
+freetype:
+  - '2'
+freexl:
+  - '2'
+fribidi:
+  - '1'
+g2clib:
+  - '1.6'
+gcab:
+  - '1'
+gdbm:
+  - '1.18'
+gdk_pixbuf:
+  - '2'
+geos:
+  - 3.10.6
+geotiff:
+  - '1.7'
+gettext:
+  - '0'
+gflags:
+  - '2.2'
+giflib:
+  - '5.2'
+gl2ps:
+  - 1.4.2
+glew:
+  - '2.2'
+glib:
+  - 2
+glog:
+  - '0.5'
+glpk:
+  - '4.65'
+glslang:
+  - '15'
+gmp:
+  - '6'
+gnupg:
+  - '2.4'
+gnutls:
+  - '3.8'
+graphite2:
+  - '1'
+gsl:
+  - '2.7'
+gst_plugins_base:
+  - 1.24
+gst_plugins_good:
+  - '1.24'
+gstreamer:
+  - '1.24'
+gstreamer_orc:
+  - '0.4'
+gtest:
+  - 1.14.0
+gtk3:
+  - '3'
+gts:
+  - '0.7'
+harfbuzz:
+  - '10'
+hdf4:
+  - 4.2.13
+hdf5:
+  - 1.14.5
+hdfeos2:
+  - 2.20
+hdfeos5:
+  - 5.1
+icu:
+  - '73'
+igraph:
+  - '0.10'
+isl:
+  - '0.22'
+jansson:
+  - '2'
+jasper:
+  - '4'
+jemalloc:
+  - '5'
+jpeg:
+  - 9e
+json_c:
+  - '0.16'
+jsoncpp:
+  - 1.9.4
+jxrlib:
+  - '1.1'
+kealib:
+  - '1.5'
+krb5:
+  - '1.21'
+lame:
+  - '3.100'
+lcms2:
+  - '2'
+leptonica:
+  - '1.82'
+lerc:
+  - '4'
+leveldb:
+  - '1.23'
+libabseil:
+  - '20250127.0'
+libaec:
+  - '1'
+libaio:
+  - '0.3'
+libapr:
+  - '1'
+libapriconv:
+  - '1'
+libaprutil:
+  - '1'
+libarchive:
+  - '3.7'
+libassuan:
+  - '3'
+libavif:
+  - '1'
+libbrotlicommon:
+  - '1.0'
+libbrotlidec:
+  - '1.0'
+libbrotlienc:
+  - '1.0'
+libclang13:
+  - '14.0'
+libclang_cpp14:
+  - '14.0'
+libcrc32c:
+  - '1.1'
+libcryptominisat:
+  - '5.6'
+libcurl:
+  - '8'
+libdap4:
+  - '3.19'
+libdb:
+  - '6.2'
+libde265:
+  - 1.0.15
+libdeflate:
+  - '1.22'
+libdrm:
+  - '2.4'
+libebm:
+  - 0.4.4
+libedit:
+  - '3.1'
+libegl:
+  - '1'
+libevent:
+  - 2.1.12
+libfaiss:
+  - 1.0.0
+libffi:
+  - '3.4'
+libflac:
+  - '1.4'
+libgcrypt:
+  - '1'
+libgd:
+  - '2.3'
+libgdal:
+  - '3.6'
+libgit2:
+  - '1.6'
+libgl:
+  - '1'
+libgles:
+  - '1'
+libglib:
+  - '2'
+libglvnd:
+  - '1'
+libglx:
+  - '1'
+libgpg_error:
+  - '1'
+libgrpc:
+  - '1.71'
+libgsasl:
+  - '1'
+libgsf:
+  - '1.14'
+libheif:
+  - '1.19'
+libiconv:
+  - '1'
+libidn2:
+  - '2'
+libjpeg_turbo:
+  - '3'
+libkml:
+  - '1.3'
+libksba:
+  - '1.6'
+liblief:
+  - '0.16'
+libllvm19:
+  - '19.1'
+libmamba:
+  - '2.0'
+libmambapy:
+  - '2.0'
+libmicrohttpd:
+  - '0.9'
+libmlir19:
+  - '19.1'
+libnetcdf:
+  - '4'
+libnghttp2:
+  - '1'
+libnsl:
+  - '2.0'
+libntlm:
+  - '1'
+libogg:
+  - '1.3'
+libopenblas:
```

Comment:
I don't think libopenblas or openblas pinnings are correct.

Reply:
The run_exports is on the major, so that does give '0' `{{ pin_subpackage("libopenblas") }}`.
Only openblas-devel has run_exports though. So it should rather be openblas and openblas-devel. 
What do yout think if I remove libopenblas and add openblas-devel pinned at '0'?

Also, once this is merged, I'll set a gha to open PRs on conda_build_config.yaml when one of these packages gets bumped to an incompatible version.

Reply:
Aha. Thanks for the explanation! That makes sense.